pkgs/development/compilers/coreclr/default.nix

```diff
@@ -27,6 +28,14 @@ stdenv.mkDerivation rec {
     sha256 = "16z58ix8kmk8csfy5qsqz8z30czhrap2vb8s8vdflmbcfnq31jcw";
   };
 
+  patches = [
+    (fetchurl {
```

Comment:
Use `fetchpatch` for patches, because the formatting of GitHub-generated patches changes over time, and `fetchpatch` removes the changing bits.

Reply:
Thanks, I assumed the formatting would be stable; will amend shortly.